pkg/controller/scyllacluster/sync_services.go

```diff
@@ -244,18 +243,28 @@ func (scc *Controller) syncServices(
 					Name:      svc.Name,
 				},
 			}
-			klog.V(2).InfoS("Evicting the Pod to replace member",
+			// TODO: Revert when kubernetes eviction is fixed in all supported versions (#732)
+			//       (https://github.com/kubernetes/kubernetes/issues/103970)
+			/*
+				klog.V(2).InfoS("Evicting the Pod to replace member",
+			*/
+			klog.V(2).InfoS("Deleting the Pod to replace member",
 				"ScyllaCluster", klog.KObj(sc),
 				"Service", klog.KObj(svc),
 				"Pod", klog.KObj(podMeta),
 			)
-			err = scc.kubeClient.CoreV1().Pods(podMeta.Namespace).Evict(ctx, &policyv1beta1.Eviction{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: podMeta.Name,
-				},
-				DeleteOptions: &metav1.DeleteOptions{
-					PropagationPolicy: &backgroundPropagationPolicy,
-				},
+			/*
```

Comment:
Not a fan of commented code, TODO comment about coming back to eviction is enough

Reply:
ok